src/libs/actions/Report.js

```diff
@@ -313,11 +313,18 @@ function fetchChatReportsByIDs(chatList, shouldRedirectIfInacessible = false) {
             Log.info('[Report] successfully fetched report data', false, {chatList});
             fetchedReports = reportSummaryList;
 
-            // If we receive a 404 response while fetching a single report, treat that report as inacessible.
+            // If we receive a 404 response while fetching a single report, treat that report as inaccessible.
             if (jsonCode === 404 && shouldRedirectIfInacessible) {
                 throw new Error(CONST.REPORT.ERROR.INACCESSIBLE_REPORT);
             }
 
+            // If the user doesn't have a report with Concierge yet, we need to create it
+            const hasConciergeChat = _.reduce(reportSummaryList, (exists, report) => exists || isConciergeChatReport(report), false);
```

Comment:
Could use `_.some()` so that we don't have to iterate over all reports and can stop at the first one we find.